src/app/home/sponsors/sponsors.component.html

```diff
@@ -0,0 +1,17 @@
+<h4 class="center-align">・{{ 'sponsors.title' | translate }}・</h4>
+<div class="row">
+  <a
+    class="col s12 m6 center-align sponsor-logo"
+    href="http://www.jalasoft.com/"
+    target="_blank"
+  >
+    <img src="assets/images/jala-logo.png" />
+  </a>
+  <a
+    class="col s12 m6 center-align sponsor-logo"
+    href="http://fundacion-jala.org"
+    target="_blank"
+  >
+    <img src="assets/images/fundacion-jala-logo.png" />
```

Comment:
We don't need to close the img tag with /> because we don't give support to xhtml. I just realized that most of our img tags are closing like this, so, this is more like a suggestion, not a must change.